src/mixins/resources/mixins.sponge.core.json

```diff
@@ -1,248 +1,249 @@
-        "world.level.levelgen.NoiseGeneratorSettingsMixin",
-        "world.level.levelgen.WorldGenSettingsMixin",
-        "world.level.levelgen.structure.LegacyStructureDataHandlerMixin",
-        "world.level.levelgen.structure.templatesystem.StructureManagerMixin",
-        "world.level.portal.PortalForcerMixin",
-        "world.level.saveddata.MapIdTrackerMixin",
-        "world.level.storage.DimensionDataStorageMixin",
-        "world.level.storage.LevelDataMixin",
-        "world.level.storage.LevelStorageSource_LevelStorageAccessMixin",
-        "world.level.storage.LevelStorageSourceMixin",
-        "world.level.storage.PlayerDataStorageMixin",
-        "world.level.storage.PrimaryLevelDataMixin",
-        "world.level.storage.ServerLevelDataMixin",
-        "world.scores.ObjectiveMixin",
-        "world.scores.PlayerTeamMixin",
-        "world.scores.ScoreboardMixin",
-        "world.scores.ScoreMixin"
-    ],
-    "client": [
-        "client.MinecraftMixin",
-        "client.multiplayer.ClientHandshakePacketListenerImplMixin",
-        "client.multiplayer.ClientLevelMixin",
-        "client.multiplayer.ClientPacketListenerMixin",
-        "client.network.chat.TranslatableComponentMixin",
-        "client.player.AbstractClientPlayerMixin",
-        "client.renderer.entity.EntityRenderDispatcherMixin",
-        "server.network.MemoryServerHandshakePacketListenerImplMixin"
-    ],
-    "server": [
-        "server.dedicated.DedicatedServerMixin"
-    ],
-    "injectors": {
-        "defaultRequire": 1,
-        "maxShiftBy": 3
-    },
-    "overwrites": {
-        "conformVisibility": true
-    }
+  "required": true,
+  "minVersion": "0.8",
+  "package": "org.spongepowered.common.mixin.core",
+  "target": "@env(DEFAULT)",
+  "compatibilityLevel": "JAVA_8",
+  "mixinPriority": 1101,
+  "mixins": [
+    "advancements.Advancement_BuilderMixin",
+    "advancements.AdvancementMixin",
+    "advancements.AdvancementProgressMixin",
+    "advancements.CriterionMixin",
+    "advancements.CriterionProgressMixin",
+    "advancements.CriterionTrigger_ListenerMixin",
+    "advancements.DisplayInfoMixin",
+    "advancements.TreeNodePositionMixin",
+    "adventure.KeyMixin",
+    "adventure.bossbar.HackyBossBarPlatformBridgeMixin",
+    "adventure.text.AbstractComponentMixin",
+    "adventure.text.format.StyleImplMixin",
+    "authlib.properties.PropertyMixin",
+    "block.BedBlockMixin",
+    "block.BlockMixin",
+    "block.DyeColorBlockPropertiesMixin",
+    "brigadier.exceptions.CommandSyntaxExceptionMixin",
+    "commands.CommandsMixin",
+    "commands.CommandSourceMixin",
+    "commands.CommandSourceStackMixin",
+    "commands.arguments.ComponentArgumentMixin",
+    "commands.arguments.DimensionArgumentMixin",
+    "commands.arguments.selector.EntitySelectorParserMixin",
+    "core.RegistryAccessMixin",
+    "core.Vec3iMixin",
+    "core.dispenser.AbstractProjectileDispenseBehaviorMixin",
+    "data.SpongeDataHolderMixin",
```

Comment:
Don't change the formatting here on a whim please.

Reply:
Not exactly sure how I managed to do that, but it should be fixed now :sweat_smile: